src/rapid_pro_to_engagement_db/sync_stats.py

```diff
@@ -13,25 +13,37 @@ class RapidProSyncEvents:
     CONTACT_NOT_IN_UUID_FILTER = "contact_not_in_uuid_filter"
     MESSAGE_ALREADY_IN_ENGAGEMENT_DB = "message_already_in_engagement_db"
     ADD_MESSAGE_TO_ENGAGEMENT_DB = "add_message_to_engagement_db"
+    RUN_VALUE_EMPTY = "run_value_empty"
 
 
-class RapidProToEngagementDBSyncStats(SyncStats):
+class FlowStats(SyncStats):
     def __init__(self):
         super().__init__({
             RapidProSyncEvents.READ_RUN_FROM_RAPID_PRO: 0,
             RapidProSyncEvents.RUN_EMPTY: 0,
             RapidProSyncEvents.RUN_CONTACT_UUID_NOT_IN_CONTACTS: 0,
             RapidProSyncEvents.UUID_FILTER_CONTACT_NOT_IN_UUID_TABLE: 0,
             RapidProSyncEvents.CONTACT_NOT_IN_UUID_FILTER: 0,
-            RapidProSyncEvents.MESSAGE_ALREADY_IN_ENGAGEMENT_DB: 0,
-            RapidProSyncEvents.ADD_MESSAGE_TO_ENGAGEMENT_DB: 0
         })
 
     def print_summary(self):
         log.info(f"Runs downloaded from Rapid Pro: {self.event_counts[RapidProSyncEvents.READ_RUN_FROM_RAPID_PRO]}")
+        log.info("The runs specified below were not processed:")
```

Comment:
I'm not convinced this is necessary. But if you think it helps please can you at least rephrase it so it's clear totals are expected. "runs specified below" implies that some run ids are about to be logged, which isn't the case.